src/category_theory/action.lean

```diff
@@ -56,25 +54,42 @@ lemma π_map (p q : action_category M X) (f : p ⟶ q) : (π M X).map f = f.val
 
 @[simp]
 lemma π_obj (p : action_category M X) : (π M X).obj p = single_obj.star M :=
-@subsingleton.elim unit _ _ _
+unit.ext
+
+variables {M X}
+/-- The canonical map `action_category M X → X`. It is given by `λ x, x.snd`, but
+  has a more explicit type. -/
+protected def back : action_category M X → X :=
+λ x, x.snd
```

Comment:
Perhaps say why you've called this `back`? (I'm failing to guess!)

Reply:
It's not a great name, I just thought of it as taking you from the action category back to X. Is there a standard name for this kind of function? Maybe `of`?

Reply:
I think `back` is fine. I would use `of` for the other direction (i.e. the `coe`).